datamodel_code_generator/__main__.py

```diff
@@ -81,234 +91,253 @@ def sig_int_handler(_: int, __: Any) -> None:  # pragma: no cover
 
 DEFAULT_ENCODING = locale.getpreferredencoding()
 
-arg_parser = ArgumentParser()
-arg_parser.add_argument(
+
+def _bold_cyan(input: str) -> str:
+    return f'\x1b[36;1m{input}\x1b[0m'
```

Comment:
Is it possible to keep the option to display in ASCII without colors to account for CICD and some special environments?

Reply:
Sure, would you prefer a new `--no-color` flag or something like that? Or I can just remove the coloring entirely

Reply:
I would expect `--no-color` :smiley:

Reply:
@koxudaxi I added a `--no-color` flag, can you check the changes in [this commit](https://github.com/koxudaxi/datamodel-code-generator/pull/1540/commits/97b323405ecb01d8eb7a5deb7ec0c3036357e04c#diff-933c022896e14daf8a2c79298b8f99fc4981915c729a41494486cd2dfc8e0336) and let me know if you're good with them?